src/Helpers/PjsConsole.js

```diff
@@ -117,19 +122,11 @@ module.exports = function PjsConsole(document) {
   };
 
   e.println = function () {
-    if(!added) {
-      document.body.appendChild(style);
-      document.body.appendChild(e.wrapper);
-      added = true;
-    }
     var args = Array.prototype.slice.call(arguments);
     args.push('<br>');
     e.print.apply(e, args);
-    if (e.BufferArray.length > e.BufferMax) {
-      e.BufferArray.splice(0, 1);
-    } else {
-      e.javaconsole.scrollTop = e.javaconsole.scrollHeight;
-    }
+    if (e.BufferArray.length > e.BufferMax) e.BufferArray.splice(0, 1);
```

Comment:
You don't like Python for sure. Boy, let's get rid of those annoying curly places whenever we can!  :P